include/podio/RNTupleWriter.h

```diff
@@ -10,11 +10,20 @@
 #include <ROOT/RNTuple.hxx>
 #include <ROOT/RNTupleModel.hxx>
 #include <ROOT/RNTupleWriter.hxx>
+#include <ROOT/RVersion.hxx>
 
 #include <string>
 #include <unordered_map>
 #include <vector>
 
+#if ROOT_VERSION_CODE < ROOT_VERSION(6, 35, 0)
+namespace ROOT {
+using REntry = ROOT::Experimental::REntry;
+using RNTupleModel = ROOT::Experimental::RNTupleModel;
+using RNTupleWriter = ROOT::Experimental::RNTupleWriter;
+} // namespace ROOT
+#endif
```

Comment:
The file format has been finalized in 6.34. Need a cleaner solution here to not pollute the ROOT namespace.